.claude/workflows/development-cycle.yaml

```
@@ -0,0 +1,144 @@
+name: "Full Development Cycle"
+description: "Complete development workflow from feature start to production deployment"
+version: "1.0"
+
+workflows:
+  feature_development:
+    name: "Feature Development Workflow"
+    description: "End-to-end feature development with quality gates"
+    triggers:
+      - pattern: ["new feature", "implement", "build"]
+      - context: ["feature branch", "development"]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)